tests/user/test_user_request_builder.py

```diff
@@ -113,9 +113,42 @@ def test_user_admin_build_alter_request_update_existing_segment_traits(self):
             update_existing_segment_traits=TestUserConstants.TEST_USER_ADDITIONAL_SEGMENT_TRAITS,
         )
         result = user_admin.alter(
-            "squidwrd", traits=TestUserConstants.TEST_ALTER_USER_CSDATA_REQUEST_TRAITS
+            "squidwrd",
+            traits=TestUserConstants.TEST_ALTER_USER_CSDATA_AND_OMVS_REQUEST_TRAITS,
         )
         self.assertEqual(
             result,
             TestUserConstants.TEST_ALTER_USER_REQUEST_UPDATE_SEGMENTS_XML,
         )
+
+    # ============================================================================
+    # Request Builder Errors
```

Comment:
Should we also have a test for alter or no?

Reply:
Since the logic is central to all security_admin objects, I didn't think it necessary to add this test for alter or even outside of user_admin, but I could add that if you would like.

Reply:
That is up to you, but you are probably right that this is good enough.